templates/projects/hello-world/files/common/models/message.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "message",
+  "properties": {
+    "title": {
```

Comment:
I am not sure if it makes sense to specify `title` and `content` properties of the `Message` model, when we are not using them at all?